hackIDE/static/hackIDE/js/custom.js

```diff
@@ -269,7 +290,13 @@ $(document).ready(function(){
 						$(".output-io-info").hide();
 						$(".compile-status").children(".value").html("--");
 						$(".error-key").html("Compile error");
-						$(".error-message").html(response.compile_status);
+
```

Comment:
I will try to explain it.

on language selector, I choose Java. If I write for example

public class TestDriver { 

a close bracket (}) is missing. if you try to compile the code the error mesaage thown is empty.

The cause of that is because ajax response variable (compile_status) contains a blank space.

Those changes are mandé un línea 294